### PR TITLE
WIP:  Address memory leaks in SST

### PR DESCRIPTION
**Pull request by eisenhauer**

### CODE REVIEW

File: source/adios2/engine/sst/SstReader.h [hunk 1]
```diff
@@ -40,7 +40,7 @@ class SstReader : public Engine
     SstReader(IO &io, const std::string &name, const Mode mode,
               MPI_Comm mpiComm);
 
-    virtual ~SstReader() = default;
+    ~SstReader();
```
[the remark this answers is not in the document]
Comment (eisenhauer) [reply source/adios2/engine/sst/SstReader.h]: Whoops.  Next update will fix.   Working on the reader side now.

File: source/adios2/engine/sst/SstWriter.h [hunk 2]
```diff
@@ -31,7 +31,7 @@ class SstWriter : public Engine
     SstWriter(IO &io, const std::string &name, const Mode mode,
               MPI_Comm mpiComm);
 
-    virtual ~SstWriter() = default;
```
[the remark this answers is not in the document]
Comment (eisenhauer) [reply source/adios2/engine/sst/SstWriter.h]: Same answer as above

**Comment by chuckatkins** [comment]
So, FWIW, the Codacy check isn't a blocker to merging.  The merge is blocked from test failures in SST now.

**Comment by eisenhauer** [comment]
Ah, I hadn't focussed on the lack of the "required" tag.  Wasn't ready to merge yet...

**Comment by chuckatkins** [comment]
Would it be too much of a hassle to move the branch to your fork instead of the main repo?  If you could, it would be much appreciated.

**Comment by eisenhauer** [comment]
Closing for resubmission...

**Pull request closed without merging**